packages/core/src/tree.js

```diff
@@ -25,6 +25,7 @@ const Tree = props => {
     onSelect,
     width,
     height,
+    styles,
```

Comment:
@liorheber In the previous PRs I removed "styles" and "classname" props bacuse I thought They are not in use.
Noe when writing the README I understood the "styles" is needed for customized styles.
What about calssname? Do we need this property?

Reply:
Which classNames? Can you link the change?
You would usually need the className prop to customize using almost any styling library - css modules, styled-components etc.

Reply:
https://github.com/kenshoo/react-tree/pull/143/files
@liorheber I tagged you in the PR

Reply:
They are for customization, however, I'm not sure it was fully implemented.
I suggest returning it and we can have a look at it at a later task when we decide to support that type of customization.